src/util/MongoBinaryDownload.js

```diff
@@ -91,13 +91,22 @@ export default class MongoBinaryDownload {
   }
 
   async checkMd5(mongoDBArchiveMd5: string, mongoDBArchive: string) {
+    if (
```

Comment:
This check could be done directly before the call of the function here : 
https://github.com/nodkz/mongodb-memory-server/blob/4f0e7ca32ff0fc90147935d9ff8d6284f755d4d6/src/util/MongoBinaryDownload.js#L88
Like that we will not have to return an undefined.

Reply:
@AJRdev this function normally returns `undefined`, so I make the same explicitly. I don't want to move it out of this function because in the future someone else can write `new MongoBinaryDownload({}).checkMd5('', '')` and MONGOMS_SKIP_MD5_CHECK would be missed.

Reply:
@dimitriylol Well I don't even see why checkMd5 should be a public member function of the MongoBinaryDownload class since it is only used internally in this class. I don't see how it would be useful for other class to call the method checkMd5 of this class, and if it is the case it should not be a member function but more an utility function not linked to a specific class.

Reply:
@AJRdev totally agree with you about moving it to a separate module. Waiting for the clarification from @nodkz

Reply:
Also better to create a new property `skipMD5` for `MongoBinaryDownloadOpts` & class. Then use it inside `checkMd5` method.

And move the reading of env variable to `constructor` if `skipMD5===undefined`:
- it makes tests more simple
- it provides a feature to disable md5 check via config options (don't forget to update README) not only via ENV vars

Reply:
@nodkz I'm sorry, I'm not clear about this. Do you want to pass it via constructor parameter *and* via ENV variable? Maybe only one source of truth is preferable. Either constructor parameter or ENV variable.

Reply:
Users may call MMS in different scenarios and may in one process run several servers with different versions. 

So it will be good if we read `skipMD5` from options firstly, and if it does not provided then read from global env variable.

Reply:
@nodkz OK, I'll make appropriate changes and add this priority to the documentation

Reply:
From readme:
```diff
const mongod = new MongoMemoryServer({
  instance: {
    port?: ?number, // by default choose any free port
    ip?: string, // by default '127.0.0.1', for binding to all IP addresses set it to `::,0.0.0.0`,
    dbName?: string, // by default generate random dbName
    dbPath?: string, // by default create in temp directory
    storageEngine?: string, // by default `ephemeralForTest`, available engines: [ 'devnull', 'ephemeralForTest', 'mmapv1', 'wiredTiger' ]
    debug?: boolean, // by default false
    replSet?: string, // by default no replica set, replica set name
    auth?: boolean, // by default `mongod` is started with '--noauth', start `mongod` with '--auth'
    args?: string[], // by default no additional arguments, any additional command line arguments for `mongod` `mongod` (ex. ['--notablescan'])
  },
  binary: {
    version?: string, // by default 'latest'
    downloadDir?: string, // by default node_modules/.cache/mongodb-memory-server/mongodb-binaries
    platform?: string, // by default os.platform()
    arch?: string, // by default os.arch()
    debug?: boolean, // by default false
+    skipMD5?: boolean, // by default false OR process.env.MONGOMS_SKIP_MD5_CHECK
  },
  debug?: boolean, // by default false
  autoStart?: boolean, // by default true
});
```

Reply:
@nodkz seems like, I finished. Please take a look